static/src/javascripts/projects/common/views/contributions-epic-single-button.html

```diff
@@ -0,0 +1,18 @@
+<div class="contributions__epic" data-component="<%=componentName%>">
+    <div>
+        <h2 class="contributions__title contributions__title--epic">
+            <%=title%></h2>
+        <p class="contributions__paragraph contributions__paragraph--epic"><%=p1%></p>
+        <p class="contributions__paragraph contributions__paragraph--epic"><%=p2%></p>
+        <p class="contributions__paragraph contributions__paragraph--epic"><%=p3%></p>
+    </div>
+
+    <div class="contributions__amount-field">
+        <div>
+            <a class="contributions__option-button contributions__contribute contributions__contribute--epic contributions__contribute--epic-member contributions__contribute--epic-member-top contributions__option-button--support-landing-page" href="<%=linkUrl1%>">
```

Comment:
For the next test, yes. But for this one, with two hours or so to get it tested and merged ready for next week, changing it isn't necessary.